client/src/main/kotlin/no/nav/su/se/bakover/client/oppgave/OppgavebeskrivelseMapper.kt

```diff
@@ -52,15 +52,13 @@ object OppgavebeskrivelseMapper {
         * Informasjonsoppgaver som må lukkes manuelt.
         * */
         KlagevedtakUtfall.TRUKKET,
-        KlagevedtakUtfall.STADFESTELSE,
-        KlagevedtakUtfall.AVVIST -> "Denne oppgaven er kun til opplysning og må lukkes manuelt."
-        /*
-        * Oppgaver som krever handling. Lukkes automatisk av `su-se-bakover`.
-        * */
-        KlagevedtakUtfall.RETUR,
+        KlagevedtakUtfall.UGUNST,
         KlagevedtakUtfall.OPPHEVET,
         KlagevedtakUtfall.MEDHOLD,
         KlagevedtakUtfall.DELVIS_MEDHOLD,
-        KlagevedtakUtfall.UGUNST -> "Klagen krever ytterligere saksbehandling. Lukking av oppgaven håndteres automatisk."
+        KlagevedtakUtfall.STADFESTELSE,
+        KlagevedtakUtfall.AVVIST -> "Denne oppgaven er kun til opplysning og må lukkes manuelt."
```

Comment:
Denne teksten gjelder kun for STADFESTELSE, AVVIST og TRUKKET. De resterende 4 trenger noe som `Klage krever ytterligere saksbehandling. Denne oppgaven må lukkes manuelt.`